reference/src/representation/function-pointers.md

```diff
@@ -0,0 +1,114 @@
+# Representation of Function Pointers
+
+### Terminology
+
+In Rust, a function pointer type, is either `fn(Args...) -> Ret`, or
+`extern "ABI" fn(Args...) -> Ret`. A function pointer is the address of a
+function, and have function pointer type.
+The pointer is implicit in the `fn` type,
+and `fn` types are implicitly `'static`.

```

Comment:
There's also `unsafe fn()` and `unsafe exterN "ABI" fn()`.

Reply:
ah, true